app/models/miq_provision_virt_workflow.rb

```diff
@@ -196,15 +196,7 @@ def validate_sysprep_field(field, values, dlg, fld, value)
     return nil unless get_value(values[field]).blank?
     "#{required_description(dlg, fld)} is required"
   end
-
-  def default_require_sysprep_enabled(_field, _values, dlg, fld, value)
-    "#{required_description(dlg, fld)} is required" if value.blank? || value == "disabled"
-  end
-
-  def default_require_sysprep_custom_spec(_field, _values, dlg, fld, value)
```

Comment:
These two methods are optional validation methods that can be applied to provisioning dialogs.  They were implemented for a user that wanted to enforce that a customization spec was selected before submitting a request.  (https://bugzilla.redhat.com/show_bug.cgi?id=MANAGEIQ-14510)  We do not enforce that restriction out-of-the-box.

We need to retain these methods.